### PR TITLE
Fix the defines for HFSTS6 enforcement policy

### PR DESCRIPTION
**Pull request by hughsie**
We were accidentally testing for the 'shutdown after 30 minutes' due to a typo, so accept that to be pragmatic or the much better immediate shutdown policy.

Noticed by Martin Fernandez <martin.fernandez@eclypsium.com> -- many thanks.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/pci-mei/fu-pci-mei-plugin.c [hunk 1]
```diff
@@ -428,8 +428,10 @@ fu_plugin_add_security_attrs_bootguard_policy(FuPlugin *plugin, FuSecurityAttrs
 		return;
 	}
 
-	/* policy must be to immediately shutdown */
-	if (self->hfsts6.fields.error_enforce_policy != ME_HFS_ENFORCEMENT_POLICY_SHUTDOWN_NOW) {
+	/* policy must be to immediately shutdown or after 30 mins -- the latter isn't ideal but
+	 * we've been testing for this accidentally for a long time now */
+	if (self->hfsts6.fields.error_enforce_policy != ME_HFS_ENFORCEMENT_POLICY_SHUTDOWN_NOW &&
```
Comment (superm1) [inline plugins/pci-mei/fu-pci-mei-plugin.c:433]: Why not just greater than 1?

Comment (hughsie) [reply plugins/pci-mei/fu-pci-mei-plugin.c:433]: I really wanted to be explicit about what we accepted.

File: plugins/pci-mei/fu-mei-common.h [hunk 2]
```diff
@@ -191,9 +191,10 @@ typedef union {
 #define ME_HFS_MODE_UNKNOWN_6 6
 #define ME_HFS_MODE_MAYBE_SPS 7
 
-#define ME_HFS_ENFORCEMENT_POLICY_NOTHING      0b00
-#define ME_HFS_ENFORCEMENT_POLICY_SHUTDOWN_TO  0b01
-#define ME_HFS_ENFORCEMENT_POLICY_SHUTDOWN_NOW 0b11
+#define ME_HFS_ENFORCEMENT_POLICY_NOTHING	  0x0
+#define ME_HFS_ENFORCEMENT_POLICY_SHUTDOWN_TO	  0x1
```
[the remark this answers is not in the document]
Comment (hughsie) [reply plugins/pci-mei/fu-mei-common.h:195]: Consensus here is that `TO` is *timeout* i.e. same as 30mins but without a specified delay.

[the remark this answers is not in the document]
Comment (mefff) [reply plugins/pci-mei/fu-mei-common.h:195]: Makes sense

**Review by superm1 — APPROVED** [review]

**Pull request merged**